content/ytt/docs/latest/lang-ref-ytt-library.md

```diff
@@ -26,6 +26,9 @@ Library module `@ytt:library` provides a way to programmatically get result of t
 # build library instance
 app1 = library.get("app")
 
+# build new copy of library with data values schema (does not mutate app1)
+app1_with_schema = app1.with_data_values_schema({"name": "default"})
```

Comment:
lets show an example that uses yamlfragment instead since above example cannot show where to put annotations.